docker-compose.ssl-manual.yml

```diff
@@ -1,5 +1,4 @@
 # This configuration adds the volumes necessary for SSL manual setup
-version: '2.1'
```

Comment:
See the discussion here for this: https://github.com/OpenDroneMap/WebODM/pull/1570

Reply:
Seen. Docker-compose's support ended in 2023. https://www.docker.com/blog/new-docker-compose-v2-and-v1-deprecation/

Also, we def. need v2 format to support things like multi-stage.

Reply:
Note that sticking with v1 I'm not sure if it makes sense for me to actively contribute here, I might fork to be able to move ahead.

Reply:
Aren't we on v2.1 currently?

My main concern (from issue #1570) is whether removing the version number definition will break the software for people still on some old version of docker-compose.

Reply:
Honestly, I'd rather not go for latest - I'd aim for supported LTS releases of everything.

Running unpatched software on the open internet is not something I'd like to do even on my home server and will not pass any form of compliance requirements.

What I am saying is, in it's current state, WebODM _is not safe for public use_.

So that's what I'd be changing, step by step.

I'm happy to think about supporting users on upgrade paths though. As there's already import/export functionality, perhaps it can be extended to do mass import/export?

Is there perhaps a way for us to have an in-sync video call or even chat to see if we can align on these issues? I'd be happy to put a considerable amount of time into it over the coming weeks.